tribe/models.py

```diff
@@ -46,9 +47,19 @@ class Meta:
         managed = False
         db_table = 'Rockstars'
 
+class ArticleText(models.Model):
+    articletextblockid = models.TextField(db_column='ArticleTextBlockId', primary_key=True)
+    articleid = models.ForeignKey('Article', models.CASCADE, db_column='ArticleId', blank=True, null=True)
+    text = models.TextField(db_column='Text', blank=True, null=True)
+
+    class Meta:
+        managed = False
+        db_table = 'ArticleTextBlocks'
+
 class Article(models.Model):
     articleid = models.AutoField(db_column='ArticleId', primary_key=True)
     rockstar = models.ForeignKey('Rockstar', models.CASCADE, db_column='RockstarId', blank=True, null=True)
+    # articletext = models.ForeignKey(ArticleText, models.CASCADE, blank=True, null=True)
```

Comment:
Where's this for?

Reply:
Removed it. I was just testing some stuff